sklearn/covariance/_empirical_covariance.py

```diff
@@ -235,7 +235,8 @@ def fit(self, X, y=None):
         return self
 
     def score(self, X_test, y=None):
-        """Compute the log-likelihood of a Gaussian data set with `self.covariance_`.
+        """Compute the log-likelihood of X_test under the Gaussian model
+         using `self.covariance_`.
```

Comment:
Can you keep it on a single line (numpydoc validation :)

Reply:
```suggestion
        """Compute the log-likelihood of `X_test` under the Gaussian model
         using `self.covariance_`.
```

Reply:
```suggestion
        """Compute the log-likelihood of `X_test` under the Gaussian model using `self.covariance_`.
```